<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendWindow</class>
 <widget class="QDialog" name="FriendWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Морской бой - Против друга</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                    stop:0 #0a2463, stop:1 #3e92cc);
    border: 3px solid #1e3d6b;
}

QLabel {
    color: #f8f8ff;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
}

QLabel#labelPlayer1, QLabel#labelPlayer2 {
    font-size: 22px;
    font-weight: bold;
    color: #f8f8ff;
    background-color: rgba(30, 61, 107, 0.7);
    border-radius: 8px;
    padding: 5px 10px;
    border: 2px solid #d4af37;
}

QLabel#labelPlayer1Ships, QLabel#labelPlayer2Ships {
    font-size: 14px;
    color: #f8f8ff;
    font-weight: bold;
}

QPushButton {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #1e3d6b, stop:1 #0a2463);
    border: 2px solid #d4af37;
    border-radius: 8px;
    padding: 8px 16px;
    font-family: &quot;Arial Rounded MT Bold&quot;;
    font-size: 14px;
    color: #f8f8ff;
}

QPushButton:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #3a5f8c, stop:1 #1e3d6b);
}

QPushButton:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #0a2463, stop:1 #1e3d6b);
}

QPushButton#btnRestart {
    min-width: 120px;
    min-height: 40px;
    font-size: 16px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #4CAF50, stop:1 #2E7D32);
    border: 2px solid #81C784;
}

QPushButton#btnRestart:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #66BB6A, stop:1 #388E3C);
}

QPushButton#btnRestart:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #2E7D32, stop:1 #1B5E20);
}

QPushButton#btnExit {
    min-width: 120px;
    min-height: 40px;
    font-size: 16px;
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #c62828, stop:1 #b71c1c);
    border: 2px solid #e57373;
}

QPushButton#btnExit:hover {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #ef5350, stop:1 #e53935);
}

QPushButton#btnExit:pressed {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #b71c1c, stop:1 #7f0000);
}

/* Стили для игровых клеток */
QPushButton {
    background-color: lightblue;
    border: 1px solid #1e3d6b;
    min-width: 30px;
    min-height: 30px;
}

QPushButton[hasShip=&quot;true&quot;] {
    background-color: #2c3e50;
}

QPushButton[hasShip=&quot;false&quot;] {
    background-color: lightblue;
}

QPushButton[hasShip=&quot;true&quot;]:disabled {
    background-color: #c62828;
}

QPushButton[hasShip=&quot;false&quot;]:disabled {
    background-color: white;
}
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="margin" stdset="0">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPlayer1">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Игрок 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPlayer2">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Игрок 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="middleLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="player1Layout">
       <item>
        <widget class="QWidget" name="gridLayoutPlayer1" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPlayer1Ships">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Корабли: 10</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="player2Layout">
       <item>
        <widget class="QWidget" name="gridLayoutPlayer2" native="true">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPlayer2Ships">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Rounded MT Bold</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Корабли: 10</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTurnInfo">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Rounded MT Bold</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ход Игрока 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRestart">
       <property name="minimumSize">
        <size>
         <width>156</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Новая игра</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="minimumSize">
        <size>
         <width>156</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
